profile/nodejs.sh

```diff
@@ -1,11 +1,23 @@
 calculate_concurrency() {
-  MEMORY_AVAILABLE=${MEMORY_AVAILABLE-$(detect_memory 512)}
-  WEB_MEMORY=${WEB_MEMORY-512}
-  WEB_CONCURRENCY=${WEB_CONCURRENCY-$((MEMORY_AVAILABLE/WEB_MEMORY))}
-  if (( WEB_CONCURRENCY < 1 )); then
-    WEB_CONCURRENCY=1
+  local memory_available=$1
+  local web_memory=$2
+  local concurrency=$WEB_CONCURRENCY
+
+  # if the user hasn't set a value for WEB_CONCURRENCY 
+  # we compute a reasonable value
+  if [[ -z "$WEB_CONCURRENCY" ]]; then
```

Comment:
No, as it stands, we're checking the global here, but your other commend would get rid of this 👍